lumigator/python/mzai/jobs/evaluator_lite/evaluator_lite/requirements.txt

```diff
@@ -0,0 +1,24 @@
+accelerate==0.33.0
+bert_score==0.3.13
+bitsandbytes==0.42.0
+click>=8.1.7
+datasets==2.20.0
+einops==0.8.0
+evaluate==0.4.2
+lm-eval==0.4.3
+loguru==0.7.2
+mistralai==0.4.2
+nltk==3.8.1
+numpy<2.0.0
+openai==1.38.0
+protobuf>=3.20.2
+pydantic-yaml>=1.2.0
+pydantic>=2.6.4
+ray[default]==2.30.0
+ruff==0.5.5
+s3fs==2024.5.0
+scipy==1.13.1
+sentencepiece==0.2.0
+transformers==4.43.4
+urllib3>=1.26.18,<2
+wandb==0.16.3
```

Comment:
I think we're not using the following deps anymore:

```
bitsandbytes
einops
lm-eval
nltk
protobuf
```

Can you check if we still need any of these?

Also, do we need to install `ruff`? And what about `wandb`? I see we're importing it in a few Python files. Maybe, we should note that this is something to remove later.

Reply:
`Einops` is a transformers/torch dependency that we had issues without at one point, `lm-eval`, we don't currently use it but we do have an entrypoint for it, so we should either remove that or completely develop it, `nltk` we are using, `protobuf` we need on the ray side, and `bitsandbytes` is also needed on the ray side for PyTorch optimization. `wandb` is a carryover from the old evaluator, so we don't need it. 

Thanks for checking!